### ISSUE TITLE
Crash after updating to 0.2.2+mc1.20.1

### ISSUE THREAD

**Issue by mpustovoi** [ThinkingStudios/RyoamicLights#19]
**Describe the bug**
On my large modpack game crashes on startup after I updated `RyoamicLights` from `0.2.1` to `0.2.2`
Possibly, it's an incompatibility. I'm not a technical specialist.

**To Reproduce**
1. For me it was just updating your mod.
2. You are unlikely to be able to repeat this. Maybe you can at least help to find out the reason of the crash.

**Expected behavior**
I expect the game to load.

**Screenshots**
Crash happens before I enter the main menu.

**Desktop (please complete the following information):**
 - OS: `Windows`
 - Minecraft `1.20.1`
 - `Forge Loader 1.20.1-47.2.23`
 - Mods: large list in `latest.log`
 - Version: `0.2.2+mc1.20.1`

**Additional context**
`latest.log` and `crash-report`: https://gist.github.com/mpustovoi/43c8ca2603f8d7f351384d9ac6c32cc4

**Comment by mpustovoi** [comment]
I see that in `crash-report` it says `java.lang.NullPointerException: Registry Object not present: snifferplus:stone_pine_leaves`. But it's `ModernFix` that sometimes causes such weird crash-reports.
However, in `latest.log` I can see different problem:
```log
[20Apr.2024 11:25:17.572] [modloading-worker-0/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Failed to create mod instance. ModID: ryoamiclights, class org.thinkingstudio.ryoamiclights.forge.RyoamicLightsForge
java.lang.IllegalArgumentException: Listener for event class net.minecraftforge.client.event.RenderLevelStageEvent takes an argument that is not a subtype of the base type interface net.minecraftforge.fml.event.IModBusEvent
	at net.minecraftforge.eventbus.EventBus.addListener(EventBus.java:249) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.addListener(EventBus.java:244) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.addListener(EventBus.java:199) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.addListener(EventBus.java:193) ~[eventbus-6.0.5.jar%2387!/:?]
	at org.thinkingstudio.ryoamiclights.forge.RyoamicLightsForge.onInitializeClient(RyoamicLightsForge.java:50) ~[RyoamicLights-forge-0.2.2+mc1.20.1.jar%23853!/:?]
	at org.thinkingstudio.ryoamiclights.forge.RyoamicLightsForge.<init>(RyoamicLightsForge.java:38) ~[RyoamicLights-forge-0.2.2+mc1.20.1.jar%23853!/:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:70) ~[javafmllanguage-1.20.1-47.2.23.jar%23946!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:124) ~[fmlcore-1.20.1-47.2.23.jar%23945!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

**Comment by zeid0ne** [comment]
same issue

**Comment by TexBlock** [comment]
fixed in 0.2.3
